### PR TITLE
Add language support

### LINKED ISSUE #150: support lang attribute for language
https://github.com/microformats/microformats2-parsing/issues/3 is the current proposal for extracting and surfacing the `lang` attribute in parsed mf2 for a page's or item's language. that proposal isn't official yet, but it's been stable for over 2y, and it's implemented in PHP and JS, so it'd be nice for m2py to support it too!

example use case: apple/itunes require the `<language>` element in podcast RSS feeds. i'd like to convert HTML `lang` to RSS `<language>` in granary.

### PR DESCRIPTION
**Pull request by angelogladding**
Add language details to embedded properties and root microformats using the following order of specificity:

* embedded properties (`class=e-* lang=..`)
* root microformats (`class=h-* lang=..`)
* document root (`<html lang=..>`)

Closes #150.

### CODE REVIEW

**Review by snarfed — COMMENTED** [review]
Awesome, thank you! This lgtm, and I appreciate the tests, but I don't actually know this codebase, so I defer to @capjamesg @sknebel @gRegorLove @kevinmarks et al.

**Review by capjamesg — APPROVED** [review]
Looks good to me! This is a concise, elegant implementation. We may want another review from someone familiar with the codebase per @snarfed's comment.

**Pull request closed without merging**
